### PR TITLE
installation-on-docker

### PR DESCRIPTION
**Pull request by kghamilton89**
# 5486

### CODE REVIEW

File: installation-on-docker.md [hunk 1]
```diff
@@ -100,26 +100,26 @@ To bind the Collector to a particular port instead of a random one, replace `--p
...
  --name=axibase-collector \
- --env COLLECTOR_USER_NAME=adm-dev \
- --env COLLECTOR_USER_PASSWORD=my\$pwd \
+ --env COLLECTOR_USER_NAME=mary.jones \
+ --env COLLECTOR_USER_PASSWORD=password \
  --env ATSD_URL=https://192.0.2.1:8443 \
  axibase/collector:latest
 ```
 
-For example, to set the maximum Java heap size, specify:
```
Comment (rodionos) [inline installation-on-docker.md]: Should it be 'To increase the maximum ...'?

File: installation-on-docker.md [hunk 2]
```diff
@@ -100,26 +100,26 @@ To bind the Collector to a particular port instead of a random one, replace `--p
...
 |`ATSD_SERVICE_PORT_TCP` | No | TCP port for [network commands](https://axibase.com/docs/atsd/api/network/). |
 |`ATSD_URL` | No | URL (`protocol://host:port`) for the Axibase Time Series Database connection.|
-|`COLLECTOR_USER_NAME` | No | User name for the [data collector](https://axibase.com/docs/atsd/administration/collector-rw-account.html) account. |
+|`COLLECTOR_USER_NAME` | No | Username for the [data collector](https://axibase.com/docs/atsd/administration/collector-rw-account.html) account. |
 |`COLLECTOR_USER_PASSWORD` | No | [Password](https://axibase.com/docs/atsd/administration/user-authentication.html#password-requirements) for the data Collector account.|
 |`DOCKER_HOSTNAME` | No | Hostname of the Docker host where Axibase Collector container is running.|
 |`JAVA_OPTS` | No| Java VM options.<br>By default the Collector starts with option `-Xmx256m` |
 
-For example, for user `adm-dev` with the password `my$pwd` sending data to ATSD at `https://192.0.2.1:8443`, specify:
+To send data to ATSD at `https://192.0.2.1:8443` as user `mary.jones` with the password `password` sending data to ATSD at , specify:
```
Comment (rodionos) [inline installation-on-docker.md]: The purpose of this example was to show how to escape special chars in the password. Lets remove this example altogether

File: installation-on-docker.md (hunk 1 above)
Comment (kghamilton89) [reply installation-on-docker.md]: changed

File: installation-on-docker.md (hunk 2 above)
Comment (kghamilton89) [reply installation-on-docker.md]: I wondered if that was why this example was included. I'll remove.

**Pull request merged**